Makefile

```diff
@@ -29,18 +29,18 @@ init-full: init install-deps
 .PHONY: init update-deps install-deps init-full
 
 # Docker
-docker-up: ## Start the docker-compose
+start: ## Start the docker-compose
 	docker-compose up
 
-docker-up-d: ## Start the docker-compose in background
+start-d: ## Start the docker-compose in background
 	docker-compose up -d
 
-docker-up-b: ## Start the docker-compose and rebuild containers
+start-b: ## Start the docker-compose and rebuild containers
 	docker-compose up --build
 
 .PHONY: docker-up docker-up-d docker-up-b
```

Comment:
If you replace the command names, also replace them in the .PHONY line.

More informations on the .PHONY : https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile

Reply:
Sorry about that, I fix it on the last commit